packaging/mgmtworker/files/opt/mgmtworker/scripts/fetch-logs

```diff
@@ -0,0 +1,117 @@
+#! /opt/mgmtworker/env/bin/python3
+import argparse
+import os
+import sys
+from tempfile import mkdtemp, NamedTemporaryFile
+
+import requests
+import shutil
+
+from cloudify.zip_utils import make_zip64_archive
+
+BASE_URL = 'https://{server}:8009/cfylogs/'
+
+
+def get_logs_dir(server, path, data_dir, log_handle, session):
+    save_path = os.path.join(data_dir, server, path)
+    url = BASE_URL.format(server=server) + path
+
+    contents = session.get(url)
+
+    os.makedirs(save_path, exist_ok=True)
```

Comment:
move this to after the `if`, no need to create the dir at all if we fail to grab the index

Reply:
Good catch.